ambari-common/src/main/python/resource_management/libraries/execution_command/cluster_settings.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+"""
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+"""
+
+__all__ = ["ClusterSettings"]
+
+class ClusterSettings(object):
+  """
+  This class maps to "/clusterSettings" in command.json which includes cluster setting information of a cluster
+  """
+
+  def __init__(self, clusterSettings):
+    self.__cluster_settings = clusterSettings
+
+  def __get_value(self, key):
+    """
+    Get corresponding value from the key
+    :param key:
+    :return: value if key exist else None
+    """
+    return self.__cluster_settings.get(key)
+
+  def is_cluster_security_enabled(self):
+    """
+    Check cluster security enabled or not
+    :return: "True" or "False" string
+    """
+    security_enabled = self.__get_value("security_enabled")
+    return True if security_enabled.lower() == "true" else False
```

Comment:
The doc comment shouldn't say the method returns a `string` when returned value is really a `bool`.  (Same issue in other methods below...)

Reply:
I will change the doc. I think the libraray should return the correct type, the original code makes the caller to call lower() and compare it with "true" or "false" string which is error prone.